content/docs/command-reference/status.md

```diff
@@ -84,6 +84,11 @@ describing the changes (described below).
     hash in the DVC-file is up to date, but there is no corresponding
     <abbr>cache</abbr> file or directory.
 
+- _update available_ means that <abbr>import stages</abbr> are not updated to
+  latest version. The orginal file or directory from where the import was done
+  is changed. The imported file can me moved to latest version by running
+  `dvc update`.
```

Comment:
```suggestion
- _update available_ means that <abbr>import stages</abbr> are outdated.
  The original file or directory imported has changed. The imported data can be
  moved to its latest version by using `dvc update`.
```

My suggestion probably needs rewrapping (80 char formatting).
> You can use an IDE plugin, and the pre-commit hook should enforce this, as you know (explained in our [docs contrib guide](https://dvc.org/doc/user-guide/contributing/docs#doc-style-guidelines-javascript-and-markdown)).

Please also install a spell checker to your IDE (typo: "orginal"). I use **Code Spell Checker** for VSCode.